packages/hash/frontend/src/pages/[account-slug]/types/entity-type/entity-type-header.tsx

```diff
@@ -0,0 +1,290 @@
+import {
+  EntityType,
+  extractBaseUri,
+  extractVersion,
+  PropertyTypeReference,
+  ValueOrArray,
+} from "@blockprotocol/type-system-web";
+import { faAsterisk } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@hashintel/hash-design-system/fontawesome-icon";
+import { Box, Container, Typography } from "@mui/material";
+import { Buffer } from "buffer/";
+import { useRouter } from "next/router";
+import { FunctionComponent, ReactElement, ReactNode, useMemo } from "react";
+import { FormProvider, useForm } from "react-hook-form";
+import { frontendUrl } from "@hashintel/hash-shared/environment";
+import { TopContextBar } from "../../../shared/top-context-bar";
+import { HashOntologyIcon } from "../../shared/hash-ontology-icon";
+import { OntologyChip } from "../../shared/ontology-chip";
+import { EditBar } from "./edit-bar";
+import {
+  EntityTypeEditorForm,
+  EntityTypeEditorPropertyData,
+} from "./form-types";
+import { useRouteNamespace } from "./use-route-namespace";
+import { mustBeVersionedUri } from "./util";
+import {
+  EntityTypeEntitiesContext,
+  useEntityTypeEntitiesContextValue,
+} from "./use-entity-type-entities";
+import { EntityTypeTabs } from "./entity-type-tabs";
+import { EntityTypeContext, useEntityTypeValue } from "./use-entity-type";
+
+const getBaseUri = (entityTypeId: string, namespace: string) =>
+  `${frontendUrl}/@${namespace}/types/entity-type/${entityTypeId}/`;
```

Comment:
The base URI is whatever URL is in the address bar, minus tab info – I'm not sure you need this

Shouldn't we already have the entity type here anyway? we can use `extractBaseUri` on its id to get this also

Reply:
I think building the Url like this is safer than from the url minus the tab info as we at least get some validation that the namespace exists (from `useNamespace`), but I'm not sure.
We don't have the entity type here yet, we use the value from this function to get it.